### PR TITLE
BuildSrc refactor

### LINKED ISSUE #199: Add buildSrc for Gradle Dependency Management
This project should take advantage of the Kotlin + buildSrc for better gradle dependency management

### PR DESCRIPTION
**Pull request by AbdulDroid**
closes #199 refactor gradle dependency management into buildSrc

### CODE REVIEW

File: trakt/build.gradle [hunk 1]
```diff
@@ -19,6 +19,29 @@ apply plugin: 'kotlin'
...
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+repositories {
+    mavenCentral()
+}
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
```
Comment (chrisbanes) [inline trakt/build.gradle]: This needs to be 1.6 for API 23 support

File: build.gradle [hunk 2]
```diff
@@ -105,11 +66,27 @@ subprojects {
...
 }
+repositories {
+    mavenCentral()
+}/*
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$Versions.kotlin_version"
+}*/
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
```
Comment (chrisbanes) [inline build.gradle]: This needs to be 1.6 for API 23 support

File: trakt/build.gradle (hunk 1 above)
Comment (AbdulDroid) [reply trakt/build.gradle]: Noted, will do that now

File: build.gradle (hunk 2 above)
Comment (AbdulDroid) [reply build.gradle]: Noted will do that now

File: trakt/build.gradle (hunk 1 above)
Comment (AbdulDroid) [reply trakt/build.gradle]: Done

File: build.gradle (hunk 2 above)
Comment (AbdulDroid) [reply build.gradle]: Done

**Comment by chrisbanes** [comment]
Thanks, I implemented something similar in #259

**Pull request closed without merging**